serveradmin/servershell/views.py

```diff
@@ -281,14 +281,25 @@ def _edit(request, server, edit_mode=False, template='edit'):  # NOQA: C901
             if server.object_id:
                 action = 'edited'
                 created = []
-                changed = [server._serialize_changes()]
+
+                changes = server._serialize_changes()
+                if len(changes.keys()) > 1:
+                    changed = [changes]
+                else:
+                    # Only object_id has been passed and we do not support
```

Comment:
Wouldn't it be easier to strip `object_id` from the changes then?
``` python
changes.pop('object_id', None)
```
Might be more reliable for situation in which `object_id` might have not been sent.

Reply:
No because this will leave me with a empty dictionary in a list: `[{}]` which will evaluate to True in the later if condition
```
>>> bool([{}])
True
```
I could check for the content in the if condition explicetly again or check for the object_id explicelty additionally to the length. We cannot change an object without sending an object_id anyways that would fail beforehand due to the failed query to get the current object.